examples/show_hids.py

```diff
@@ -3,8 +3,27 @@
 
 #
 """
-Show all HID devices information
+Show all HID devices' information
+
+Written By Rene Aguirre (https://github.com/rene-aguirre) ~2016
+Updated by Gabriel Staples (https://www.ElectricRCAircraftGuy.com) 2 Feb 2017 
```

Comment:
Somehow this address doesn't work with chrome, I think your site doesn't support https:// and Chrome doesn't like it. Is it just http:// ?